docs/blog/write-the-docs-2018-kickoff.rst

```diff
@@ -0,0 +1,77 @@
+.. post:: Jan 20, 2018
+   :tags: community updates, core team
+   :author: Mikey Ariel
+
+Write the Docs 2018 Kick-Off
+============================
+
+Greetings, global documentarians!
+
+Mikey here, writing from the deep dark cold European winter, but looking forward to spring and summer, full of sunshine and docs!
+Following Eric's winter-warming `2017 Stats <http://www.writethedocs.org/blog/write-the-docs-2017-stats/>`_ post, I'd like to highlight some of the activities, changes, and experiments that you can expect in 2018.
+
+These efforts are a part of optimizing our work in supporting and growing the community, and the also aim to keep things fresh and exciting for our documentarians.
+
+We hope that you'll enjoy the shiny new things and of course, if you'd like to contribute to any of these efforts or have experiments that you'd like to pursue, please write to us at support@writethedocs.org and let us know!
+
+So, new year - new...
+
+Core Team
+---------
+
+Since the original Write the Docs conference was held in 2012, the folks behind the scenes have contributed their time, efforts, and talent on a volunteer basis, with tremendous results.
+
+Over the years, the community grew to a point where we needed to examine its long-term needs and develop a system that will support the community without burning out the people who are filling key global roles.
+
+Last year we decided to try setting up a formalized a small, part-time paid staff with the goal of consolidating various roles and responsibilities that could be handled globally, such as web administration, swag coordination, staff wrangling, and meetup overseeing.
+
+So far the experiment has been successful, and the core team now includes the following dedicated documentarians:
+
+- Eric Holscher: Co-founder, head of North America conferences, global sponsorship coordinator
+- Mikey Ariel: Head of Europe conferences, global swag coordinator, communication chair
+- Samuel Wright: Head of support, Webmaster/GitHubmaster, meetup overseer, speaker coordinator (Europe)
+- Kelly O'Brien: Head of newsletter, job board coordinator, speaker coordinator (North America)
```

Comment:
We should probably tone down some of the language here, since we have some dedicated teams to handle some of these tasks, and we're just the "official point of contact", eg. for meetups. I think "coordinator" is good language we've used in the past to not imply ownership.